jsettlers.common/src/main/resources/jsettlers/common/buildings/building.dtd

```diff
@@ -1,5 +1,37 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
+<!--
+Create new Building (Author: MarviMarv, 13.01.2021)
+	1.	Add Building name to enum “EBuildingType” (uppercase name)
+	2.	Add Building to related category in enum “EBuildingsCategory”
+	3.	Add Building to enum “EOriginalMapBuildingType” (seems to be deprecated though)
+	4.	Add CASE for method “createBuilding” in class “Building” depending on the actual type the building is related to (specific building class), you may need to implement a new class
+	5.	Create xml at path “jsettlers.common\src\main\resources\jsettlers\common\buildings\egyptian\brewery.xml” with the same name you have used for the “EBuildingType” from step 1 (lowercase name)
```

Comment:
The path should be jsettlers.common\src\main\resources\jsettlers\common\buildings\<nation>\<enum constant from in lowercase EBuildingType>.xml and not specifically the egyptian brewery.